jscomp/build_tests/utils.js

```diff
@@ -0,0 +1,51 @@
+const child_process = require("child_process");
+
+const signals = {
+  SIGINT: 2,
+  SIGQUIT: 3,
+  SIGKILL: 9,
+  SIGTERM: 15,
+};
+
+/**
```

Comment:
It would be great if you could add a comment here explaining why we need this function instead of `spawnSync` / what issue it fixes.

Reply:
It is an asynchronous version of spawnSync, with no special behavior. But preferred than `spawnSync` because `spawnSync` is unstable for unknown reason.

I think it would be better to add some explanation onto `execReScript` util you suggested.

Reply:
Yes, we can do it there, too.